src/components/TreeGrid/TreeGrid.tsx

```diff
@@ -77,6 +77,9 @@ export class TreeGrid extends React.PureComponent<ITreeGridProps, ITreeGridState
         };
         expandedColumns.push(replacementFirstColumn);
         for (let i = 1; i < columns.length; i++) {
+            if (columns[i].cellFormatter == null && columns[i].dataType === DataTypeEnum.Boolean) {

```

Comment:
won't this change the original columns? do we want that?

Reply:
You're right, this would cause rerenders. Spreading the columns to a new object now